LSP-pyright.sublime-settings

```diff
@@ -42,9 +42,9 @@
 		//          "pyright.dev_environment.gdb.binary" controls which exectuable to call to invoke GDB.
 		"pyright.dev_environment": "",
 		// When the predefined setup is "blender", invoke this binary to query the additional search paths.
-		"pyright.dev_environment.blender.binary": "blender",
```

Comment:
Using `pyright.dev_environment.blender.binary` as key causes it to be overwritten by `pyright.dev_environment` due to how the DottedDoct class works... I went with `pyright.dev_environment_blender.binary` (note the underscore).

Reply:
oops that's embarrassing...